nvflare/private/fed/server/server_engine_internal_spec.py

```diff
@@ -52,12 +52,13 @@ def get_staging_path_of_app(self, app_name: str) -> str:
         """
         pass
 
-    def deploy_app_to_server(self, app_name: str, app_staging_path: str) -> str:
+    def deploy_app_to_server(self, run_destination: str, app_name: str, app_staging_path: str) -> str:
```

Comment:
Just at note.
For the "Specs", it might be useful to declare it as ABC and use the decorator "@abstractmethod" to require its child class to implement those methods.

And the IDE will report error if those methods are not implemented.

Reply:
done